src/common/util/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ add_library(${TARGET_NAME} STATIC ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 add_library(openvino::util ALIAS ${TARGET_NAME})
 
 target_link_libraries(${TARGET_NAME} PRIVATE ${CMAKE_DL_LIBS})
+if (WIN32)
+    target_link_libraries(${TARGET_NAME} PRIVATE Shlwapi)
+endif()
```

Comment:
why this library is needed?

Reply:
I use `PathIsRelative()` method from that library in order to determine path is absolute or not. We may use `std::experimental::filesystem::path is_absolute()` from C++14 or write custom implementation (but Windows have several cases like "X:\", "\\", "\\?\" etc. to be handled)

Reply:
For VS we use C++14 can we try touse API from 14 standard? Does it work?

Reply:
@ilyachur 
```
fatal error C1189: #error:  The <experimental/filesystem> header providing std::experimental::filesystem i
s deprecated by Microsoft and will be REMOVED. It is superseded by the C++17 <filesystem> header providing std::filesystem. You can define _SILENCE_EXPERIMENTAL_FILESYSTEM_DEPRECATION_WARNING to acknowledge that you have received this warning.
```